src/OpenLoco/Paint/PaintVehicle.cpp

```diff
@@ -115,7 +115,7 @@ namespace OpenLoco::Paint
                         // Ghosts don't cast shadows
                         return;
                     }
-                    session.setItemType(Ui::ViewportInteraction::InteractionItem::t_0);
+                    session.setItemType(Ui::ViewportInteraction::InteractionItem::noInteraction);
```

Comment:
It looks like this is how ghost vehicles are set to not interact. I think it would be better if we handled the disable scenery in the same way but unfortunately we have not implemented the painting functions for them. Its worth keeping this in mind though and perhaps changing this in the future.

Reply:
I'll leave a TODO note.